clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -139,15 +142,37 @@ static cl::list<std::string> Inputs(cl::Positional, cl::OneOrMore,
                                     cl::desc("<input files>"),
                                     cl::cat(ClangOffloadWrapperCategory));
 
+// CLI options for device image compression.
+// TODO: Turn off this option by default.
+static cl::opt<bool>
+    SYCLCompressDevImg("sycl-compress-dev-imgs", cl::init(true), cl::Optional,
```

Comment:
making sycl-specific changes here is not ideal IMO.

i wonder, could we instead do the compression as part of `sycl-post-link` and track the compression/compression type as an image property. 

maybe you already considered this and there's a reason to not do that, let me know :)

Reply:
I've renamed new CLI options added to `clang-offload-wrapper` to be sycl-agnostic: similar to the what's done in upstream `clang-offload-bundler` (https://github.com/intel/llvm/blob/sycl/clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp#L180).
Also, I now link LLVMSupport library with `clang-offload-wrapper` to do the compression.